script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -1054,7 +1054,7 @@ What made you suddenly think that I'm a transfer student?"
 [sha:70bdc8094db70f9f55cfad28815c92703df90b6e]{
 -- Page 140, Offset 828.
 -- 今までは親戚の家にいたんですけど、今日からもとの家に戻るだけなんですよ」
-I've been living with my relatives up til now, but from today on I'll be back at my old house."
+I've been living with my relatives up till now, but from today on I'll be back at my old house."
```

Comment:
One `l`
https://www.merriam-webster.com/words-at-play/should-you-use-until-or-till-or-til